pyphare/pyphare/pharesee/particles.py

```diff
@@ -67,6 +67,7 @@ def add(self, particles):
         self.v        = np.concatenate((self.v, particles.v))
         self.charges  = np.concatenate((self.charges, particles.charges))
         self.weights  = np.concatenate((self.weights, particles.weights))
+        self._x = None
```

Comment:
does it make sense to move this to outside/before the if 
```
https://github.com/PHAREHUB/PHARE/blob/master/pyphare/pyphare/pharesee/particles.py#L29
```

Reply:
I don't see why not. Does it impact proposed changes or is it unrelated?

Reply:
kinda related, depends on whether this change is to make sure the attribute exists, or just to "None" it

either way, we'll probably end up with something like 

```
def _reset(self):
        self._x = None
        self._y = None
        self._z = None
```

given how it's set to "None" in more than one place